### ISSUE TITLE
Come up with nice UX for Commento commenting

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#6200]
Loading the "Commento" comment system without any customizations looks kind of terrible:

![image](https://user-images.githubusercontent.com/177243/107691467-5134b100-6c60-11eb-8ce2-28269092fc03.png)

This definitely needs some custom styling so it's on-brand and works with our styleguide.

**Comment by Pomax** [comment]
Unblocking all the CSP-blocked resources gives a slightly nicer look:

![image](https://user-images.githubusercontent.com/177243/107702133-a8418280-6c6e-11eb-8205-116cb34b12f7.png)

Is this good enough for us to turn it on, or do we want to do some finessing first still?

**Comment by Pomax** [comment]
Commento docs for customization (although it's not a lot of docs...) https://docs.commento.io/configuration/frontend/#configuration-settings

**Comment by Pomax** [comment]
It's a bit hard to tell the comments from the update content at the moment, so we may need to give this a heading and possibly accentuate it/move it around?

Current look:

![image](https://user-images.githubusercontent.com/177243/107719432-16953d80-6c8d-11eb-939b-9a9dae4628ae.png)

**Comment by Pomax** [comment]
moved with a stippled heading, but that might be too busy:

![screenshot 607](https://user-images.githubusercontent.com/177243/107719462-27de4a00-6c8d-11eb-85f3-5b291e915aa1.png)
:

**Comment by Pomax** [comment]
moved without stippling, but still with heading:

![screenshot 608](https://user-images.githubusercontent.com/177243/107719520-42182800-6c8d-11eb-8ce7-d4ad12fa3461.png)

**Comment by sabrinang** [comment]
Is it ok to refer to what is on https://foundation.mofostaging.net/en/privacynotincluded too (assuming this is the latest)? Does "powered by Commento" need to be present also?

**Comment by kristinashu** [comment]
Thank you @sabrinang! It is actually already on the live site https://foundation.mozilla.org/en/privacynotincluded/lex/

**Comment by sabrinang** [comment]
Are these states of Commento customizable too? _(sign in popover, comment box, comments etc.)_ :
<img width="818" alt="image" src="https://user-images.githubusercontent.com/2492510/107804380-adf3a280-6d18-11eb-836a-4ed78f18953c.png">
<img width="820" alt="image" src="https://user-images.githubusercontent.com/2492510/107804701-12aefd00-6d19-11eb-9040-403c373c9de2.png">
<img width="812" alt="image" src="https://user-images.githubusercontent.com/2492510/107805231-e8aa0a80-6d19-11eb-923d-10f377b530b8.png">
<img width="823" alt="image" src="https://user-images.githubusercontent.com/2492510/107805117-be584d00-6d19-11eb-919d-f52ead321ba3.png">

**Comment by sabrinang** [comment]
I iterated on an update to the area above footer section: https://share.goabstract.com/faaed37c-c73f-410d-bee3-4ce10e46766a?sha=9ad7fdcb5dff1b98d2e1f28bc51d607c59614261

- aligned with column text width with sections above
- used all h3 headings to match
- consider label change 'updates' to 'related news' instead (might make more sense if these are external articles)
- comments relocated at end as user content length varies 
- UI updates to commento to match type/color/buttons 

I can continue with all the other Commento states next week when we know what is customizable.

**Comment by kristinashu** [comment]
Oooh that looks really nice Sabs! Hopefully it will all be possible to implement.

**Comment by Pomax** [comment]
they might be, but the immediate concern is "can we add anything that lets us keep this turned on or not" before the weekend, since valentine's day is this Sunday.

**Comment by kristinashu** [comment]
@sabrinang @Pomax are you able to touch base with each to sort this out?

**Comment by sabrinang** [comment]
I think @Pomax is pto today — let's touch base tomorrow when you get the chance to look into if the states are customizable or not? I can continue iterating based on your findings and make corresponding mobile screens too.

**Comment by sabrinang** [comment]
@pomax will this Commento styling also be applied to foundation pages like blog that also use Commento?

**Comment by Pomax** [comment]
We should be able to control the overrides "per template" so as long as we save the different styles to different .css files for PNI vs. blogs, we _should_ be able to apply PNI only on product pages, and blog only on blog pages.

**Comment by Pomax** [comment]
Also, looking at https://cdn.commento.io/css/commento.css it seems like we should be able to override everything, but only in terms of styling, we have no control over the actual dom nodes/text/etc. 

Unrolled CSS that we should be able to set overrides for: https://gist.github.com/Pomax/aefc9367942e375235733be4b2e9bb3b

**Comment by sabrinang** [comment]
I mocked up some addition commento states on desktop, see abstract here: https://share.goabstract.com/a22a0eba-7b1e-4e75-aa83-81760737e246 

Examples of changes are mainly to follow our brand typography, buttons and colours:
<img width="1521" alt="image" src="https://user-images.githubusercontent.com/2492510/108289667-ab8faf00-7143-11eb-811b-b581b2c99ff0.png">

@Pomax What do you think about these type of changes? Let me know if I should proceed to make mobile/additional mockups etc.

**Comment by Pomax** [comment]
We can see how far we can get with these, first. Are we adding a heading "comments" above the comment section? (because that is not currently the case)

**Comment by sabrinang** [comment]
Sounds good @Pomax and yes the "Comments" heading is added in the proposed design.

**Comment by Pomax** [comment]
It turns out there is no clean way (at least, not documented) to do this development work locally, I've reached out to Commento to hopefully understand how we can write custom CSS and test it without having to push anything to production before it'll work... =(

**Comment by sabrinang** [comment]
Ah okay thanks for looking into this and for the update Pomax

**Comment by kristinashu** [comment]
I think design has done everything they can do so I'm going to remove Sabs and I from this ticket. Please re-assign or open a follow-up ticket if you any additional design is needed.

**Comment by Pomax** [comment]
I've filed #6338 to cover the heading, which is out content, and will move this issue into next sprint as it depends on commento getting back to us - and our decision on whether or not to stick with commento, too.

**Comment by jamilasnell** [comment]
Since we are going with Coral now, we are closing this ticket as it is no longer relevant
